webtau-cli-groovy/src/main/groovy/com/twosigma/webtau/cli/CliExtension.groovy

```diff
@@ -18,5 +18,34 @@
 
 package com.twosigma.webtau.cli
 
+import com.twosigma.webtau.cli.expectation.CliExitCode
+import com.twosigma.webtau.cli.expectation.CliOutput
+import com.twosigma.webtau.cli.expectation.CliValidationExitCodeOutputHandler
+
 class CliExtension {
+    static void run(Cli cli, String command, Closure handler) {
+        cli.run(command, closureToHttpResponseValidator(handler))
+    }
+
+    static void run(Cli cli, String command, ProcessEnv env, Closure handler) {
+        cli.run(command, env, closureToHttpResponseValidator(handler))
+    }
+
+    private static CliValidationExitCodeOutputHandler closureToHttpResponseValidator(Closure validation) {
```

Comment:
Doesn't seem like the correct name for this fuction given that it does not return a `HttpResponseValidator`.

Reply:
oh copy and paste